.github/ISSUE_TEMPLATE/support_request.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Support request
+about: Get help troubleshooting an issue
+title: ''
+labels: 'customer support'
+assignees: ''
+
+---
+The issue tracker is not for support requests. 
+
+If you would like support with Outline Manager or Outline Client, please review the help articles on [support.getoutline.org](https://support.getoutline.org/), and [contact us](https://support.getoutline.org/s/contactsupport) if you need further assistance. Developers, please post your questions on the SDK [Discussion board](https://github.com/Jigsaw-Code/outline-sdk/discussions).
```

Comment:
Since there are 3 separate places to direct people, I couldn't think of a single link that would suffice. What did you have in mind?

Reply:
Since these are the set of template changes we agreed to in the doc, are you comfortable with us implementing them as written and then we can always iterate on them going forward?